analysis/webservice/apidocs/openapi.yml

```diff
@@ -671,12 +673,19 @@ components:
           type: string
         source:
           type: string
-        analysed_sst:
-          type: number # TODO Fix this
         matches:
           type: array
           items:
-            $ref: '#/components/schemas/DomsPoint'
+            $ref: '#/components/schemas/DomsPointSecondary'
```

Comment:
It looks like this is a circular reference since DomsPointSecondary references DomsPointBase and DomsPointBase references DomsPointSecondary. Perhaps it's meant to be DomsDataPoint?

Reply:
Thanks @joshgarde! I believe I've addressed the circular reference by moving the `matches` property from `DomsPointBase` to `DomsPointPrimary`